sys/include/net/gnrc/lwmac/lwmac.h

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_lwmac Simplest possible MAC layer
+ * @ingroup     net_gnrc
+ * @brief       Lightweight MAC protocol that allows for duty cycling to save
+ *              energy.
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the LWMAC protocol
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GNRC_LWMAC_H
```

Comment:
`NET_GNRC_LWMAC_LWMAC_H` according to [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#header-guards) (maybe move it to `net/gnrc/lwmac.h` to avoid double `LWMAC`)